src/main/docs/guide/views/templates/jte.adoc

```diff
@@ -26,3 +26,7 @@ When 'dynamic' is enabled (see below), JTE will load templates from the project
 The properties used can be customized by overriding the values of:
 
 include::{includedir}configurationProperties/io.micronaut.views.jte.JteViewsRendererConfigurationProperties.adoc[]
+
+== .kte (Kotlin) templates
+
+To use `.kte` (Kotlin) templates include the dependency https://search.maven.org/artifact/gg.jte/jte-kotlin[gg.jte:jte-kotlin].
```

Comment:
If it's `managed` we could use

```
dependency:gg.jte:jte-kotlin[gradleScope="implementation"]
```

here, as it will be defined in the POM?

Reply:
done via https://github.com/micronaut-projects/micronaut-views/pull/453/commits/a892c3789e28ac781180641d587e69860144c8e3